templates/contact-us.html

```diff
@@ -81,7 +81,7 @@ <h2 class="p-heading--3">
       <div class="row">
         <div class="col-4 p-card" style="border-style: none;">
           <h3 class="p-card__title p-heading--four u-no-margin--top">Canonical Group Limited</h3>
-          <p><small>The Office Group<br>St Dunstans House<br>4th floor<br>201 Borough High St<br>London<br>SE1 1JA</small></p>
+          <p><small>2nd Floor, <br/>Clarendon House, <br/>Victoria Street, <br/>Douglas IM1 2LN, <br/>Isle of Man</small></p>
           <p><a target="_blank" href="https://goo.gl/maps/5H9Z1wYTQ7mSiw277"><small>View map&nbsp;&rsaquo;</small></a></p>
```

Comment:
We should update the google maps link to match the new address

Reply:
Not sure, as the map is the location of the offices not the registered address.

Reply:
Ah right, understood